### ISSUE TITLE
Performance issues - unsteady framerate, freezes in certain areas

### ISSUE THREAD

**Issue by MarioSMB** [TeamTwilight/twilightforest#1607]
### Forge Version

40.1.73

### Twilight Forest Version

4.1.1220

### Steps to Reproduce

Walk around, particularly around mushrooms and such.

### What You Expected

A smooth steady framerate.

### What Happened Instead

Lag spikes and game freezes.

### Additional Details

Here is a Spark log from me walking back and forth between one of these "lag spots" - they seem to be in specific locations around the world: https://spark.lucko.me/kexc5cvbnE  
They are reproducible in new worlds, just by finding any place that will freeze the game (they are common!). Tested with a clean environment and only Twilight Forest installed.

I've spent the past few hours trying to debug massive framerate drops, those seem to be exacerbated by Serene Seasons, but the freezing does exist in the Twilight Forest mod itself, with and without mods like Starlight to boost performance.  
Unsure if this is as bad on servers, as my current testing setup is singleplayer only.

### Please Read and Confirm The Following

- [X] I have confirmed this bug can be replicated without the use of Optifine.
- [X] I have confirmed the details provided in this report are concise as possible and does not contained vague information (ie. Versions are properly recorded, answers to questions are clear).
- [X] I have confirmed this issue is unique and has not been reported already.

**Comment by wendall911** [comment]
Same issue here, especially if playing with mods that add additional mobs ... to the point that the game is unplayable. https://spark.lucko.me/o5DI6BmPVO

**Comment by GizmoTheMoonPig** [comment]
I found a few performance issue spots during 1.19 that I fixed. One was structure spawning and another was the portal advancement lock thing. Both have much smaller footprints now. I havent really seen anything that causes lag quite like those did but i'll keep an eye out. The mod seems to run a lot better in 1.19 anyway, I always recommend updating to the latest. Considering 1.18 is no longer supported anyway im gonna consider this safe to close
